stdlib/hashtbl.mli

```diff
@@ -212,6 +212,32 @@ val stats : ('a, 'b) t -> statistics
    buckets by size.
    @since 4.00.0 *)
 
+(** {6 Iterators} *)
+
+val to_iter : ('a,'b) t -> ('a * 'b) Iter.t
+(** Iterate on the whole table, in unspecified order
+        @since NEXT_RELEASE *)
```

Comment:
The behavior when the the table is mutated during the traversal should be specified or explicitly left undefined.